app/views/users/new.html.haml

```diff
@@ -12,6 +12,9 @@
               Sign up with Github
           %span.or or
           = form_for @user, url: signup_path do |f|
```

Comment:
if you use `simple_form_for @user` here you should have errors for your inputs displayed if there is a problem. Do that.